docs/wallets/decrediton/upgrading-decrediton.md

```diff
@@ -0,0 +1,16 @@
+# Upgrading Decrediton
+
```

Comment:
Could you insert a page-break here `---` as per the other pages.

Reply:
Done